R/02_importHelper.R

```diff
@@ -41,7 +41,30 @@ createGraphFromUpload <- function(graph_list) {
 updateGraph <- function(graph, uploadedGraph) {
   observe({
     req(uploadedGraph())
+    logDebug("Update graph")
```

Comment:
Just wondering, is there a reason why we use logDebug and not something else like info or similar. Just asking because debug always sounds like logging for development. Or do we want users that use the app locally to see the debug messages?